src/main/kotlin/no/nav/medlemskap/regler/common/Personfakta.kt

```diff
@@ -139,14 +139,30 @@ class Personfakta(private val datagrunnlag: Datagrunnlag) {
 
     fun hentBrukerinputArbeidUtenforNorge(): Boolean = datagrunnlag.brukerinput.arbeidUtenforNorge
 
-    fun hentBrukerLandskodeInnenfor12Mnd(): List<Adresse> {
+    fun hentBrukersBostedsadresseLandskodeInnenfor12Mnd(): List<Adresse> {
         return datagrunnlag.personhistorikk.bostedsadresser.filter {
             it.landkode == "NOR"
                     && (it.tom?.isAfter(datohjelper.kontrollPeriodeForNorskAdresse().fom) ?: true)
                     && it.fom!!.isBefore(datohjelper.kontrollPeriodeForNorskAdresse().tom)
         }
     }
 
+    fun hentBrukersPostadresseLandskodeInnenfor12Mnd(): List<Adresse> {
+        return datagrunnlag.personhistorikk.postadresser.filter {
+            it.landkode == "NOR"
+                    && (it.tom?.isAfter(datohjelper.kontrollPeriodeForNorskAdresse().fom) ?: true)
+                    && it.fom!!.isBefore(datohjelper.kontrollPeriodeForNorskAdresse().tom)
+        }
+
+    }
+
+    fun sjekkBrukersPostadresseOgBostedsadresseLandskode(): Boolean {
+        if (hentBrukersBostedsadresseLandskodeInnenfor12Mnd().isEmpty() && hentBrukersPostadresseLandskodeInnenfor12Mnd().isEmpty()) {
```

Comment:
Kotlin har noen extension functions på list som kunne vært nyttige her. I stedet for å bruke filter og så isempty kunne du vurdert å bruke all/none/any metodene på list.